src/scripts.js

```diff
@@ -21,42 +19,62 @@ const secondPlaceCard = document.querySelector(".second-place")
 const thirdPlaceCard = document.querySelector(".third-place")
 const friendsList = document.querySelector(".cards-wrap-friends")
 const userImage = document.querySelector(".user-image")
+const userSelect = document.querySelector(".user-drop")
+const userDate = document.querySelector("#user-date")
+const submit = document.querySelector(".find-date-button")
 
+let chart1
```

Comment:
I know the reason behind having these global variables besides the query selectors I just hope the instructors don't mind having extra global variables. I am thinking also on changing these variables to be `const`